networking/simple-webserver.c

```diff
@@ -0,0 +1,48 @@
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <netinet/in.h>
+#include <string.h>
+
+/*
+  This is a very simple implementation of a web server using C. It is my first contributuion to pdhoot's repository in GitHub.
+  Creates a socket and returns a simple HTML response.
+*/
+int main(int argc, char const *argv[]) {
+        char response[] = "HTTP/1.1 200 OK\r\n\r\n"
+                          "<html>\n"
+                          " <body>\n"
+                          "   <h1>It works!</h1>\n"
+                          " </body>\n"
+                          "</html>\n";
+        int sock_desc, c, accept_conn;
+        struct sockaddr_in server, client;
+        sock_desc = socket(AF_INET, SOCK_STREAM, 0);
+        if (sock_desc == -1) {
+                printf("Error creating the socket.\n");
+        }
+        server.sin_family = AF_INET; // socket family
+        server.sin_addr.s_addr = INADDR_ANY;
+        server.sin_port = htons(8081); // port
+        if (bind(sock_desc, (const struct sockaddr *) &server, sizeof(server)) < 0) {
+                printf("Error binding the socket... Is the port being used already?\n");
+        } else {
+                printf("Bind completed. Waiting for incoming connections...\n");
+                listen(sock_desc, 16);
+        }
+        // We will start accepting connections now.
+
+        accept_conn = accept(sock_desc, (struct sockaddr *) &client, (socklen_t*) &c);
```

Comment:
This code handles only one connection and then exits. Whereas, a server should be able to handle multiple connections. A while loop above this line should do the work.